tests/functional/monitoring/conftest.py

```
@@ -1213,3 +1213,102 @@ def teardown():
+    """
+    is_provider_mode = config.multicluster
+    client_name = None
+
+    if is_provider_mode:
+        logger.info("Provider mode detected, getting consumer info")
+        with config.RunWithFirstConsumerConfigContextIfAvailable():
+            cluster_id = exec_cmd(
+                "oc get clusterversion version -o jsonpath='{.spec.clusterID}'"
+            ).stdout.decode("utf-8")
+            client_name = f"storageconsumer-{cluster_id}"
+        logger.info(f"Consumer client name: {client_name}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)